helm/hello-world/values.yaml

```diff
@@ -1,111 +1,555 @@
-    # kubernetes.io/tls-acme: "true"
+
+    # @schema
+    # required: false
+    # @schema
     cert-manager.io/cluster-issuer: letsencrypt-giantswarm
+
+  # @schema
+  # title: Hosts
+  # type: array
+  # required: true
+  # items:
+  #   type: object
+  #   properties:
+  #     host:
+  #       type: string
+  #       format: hostname
+  #       description: Hostname to expose the service for.
+  #     paths:
+  #       type: array
+  #       items:
+  #         type: object
+  #         properties:
+  #           path:
+  #             type: string
+  #           pathType:
+  #             type: string
+  #             enum:
+  #             - Prefix
+  #             - Exact
+  # @schema
+  # You will have to overwrite the host value to make it match your cluster's ingress subdomain.
   hosts:
   - host: hello.cluster.provider.gigantic.io
     paths:
     - path: /
       pathType: Prefix
+
+  # @schema
+  # title: Transport layer security (TLS)
+  # type: array
+  # items:
+  #   type: object
+  #   properties:
+  #     secretName:
+  #       type: string
+  #       description: Name of the secret containing the TLS server certificate.
+  #     hosts:
+  #       type: array
+  #       items:
+  #         type: string
+  #         format: hostname
+  #       description: Hostnames to use the TLS server certificate for.
+  # @schema
+  # You will have to overwrite the hosts values to make it match your cluster's ingress subdomain, matching the names used as ingress hosts.
   tls:
   - secretName: hello-world-tls
     hosts:
     - hello.cluster.provider.gigantic.io
 
+# @schema
+# title: Resource requests and limits
+# type: object
+# @schema
+# This is directly injected as the main pods's resource requests and limits.
 resources:
+
+  # @schema
+  # title: Requests
+  # type: object
+  # @schema
   requests:
-    cpu: 200m
-    memory: 50Mi
+
+    # @schema
+    # title: CPU
+    # type: string
+    # examples:
+    #   - 100m
+    #   - "1"
+    # @schema
+    cpu: 10m
+
+    # @schema
+    # title: Memory
+    # type: string
+    # examples:
+    #   - 100M
+    #   - "1Gi"
+    # @schema
+    memory: 50M
+
+  # @schema
+  # title: Limits
+  # type: object
+  # @schema
   limits:
-    cpu: 400m
-    memory: 100Mi
 
+    # @schema
+    # title: CPU
+    # type: string
+    # @schema
+    cpu: 50m
+
+    # @schema
+    # title: Memory
+    # type: string
+    # @schema
+    memory: 100M
+
+# @schema
+# title: Automatic scaling
+# type: object
+# @schema
 autoscaling:
+
+  # @schema
+  # title: Enabled
+  # type: boolean
+  # @schema
+  # If set to false, no HorizontalPodAutoscaler resource will be created.
   enabled: true
-  minReplicas: 1
-  maxReplicas: 100
+
+  # @schema
+  # title: Min replicas
+  # type: integer
+  # minimum: 1
+  # @schema
+  minReplicas: 3
+
+  # @schema
+  # title: Max replicas
+  # type: integer
+  # minimum: 1
+  # @schema
+  # Make sure to set a value that is higher than the minReplicas value.
+  maxReplicas: 10
```

Comment:
Please do not decrease this. This app has also been used for load tests in the past, at least from me, and I therefore increased it to 100 at some point.